File: Games/Guess A Number/Gan.py
import random

guesses = 5
secretNumber = random.randint(1,10)    
loop = True
x = 0
while x != secretNumber:
    while loop:
        try:
            print(f"You have {guesses} guesses left.")
            x = int(input("Number: "))
            loop = False
        except ValueError:
            print("Please insert an integer")
    if guesses >= 1:
        guesses -= 1
        if x > secretNumber:
            print("The number you guessed is bigger than the correct number!")
            loop = True
        elif x < secretNumber:
            print("The number you guessed is less than the correct number!")
            loop = True
        else:
            print("You got the number!")
    else:
        print("You ran out of guesses!")
        x = secretNumber

File: Games/Guess A Number/gan.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>
int main(){
    srand(time(NULL));
    int r = rand() % 9;
    int secretNumber = r+1;
    int guess;
    int guesses = 5;
    int loop = 1;
    while(loop == 1){
        printf("\nYou have %d", guesses);
        printf(" guesses left!\n");
        printf("Enter a number between 1 and 10: ");
        scanf("%d", &guess);
        guesses -= 1;
        if(guesses > 0){
            if (guess == secretNumber){printf("You guessed the number!"); loop = 0;}
            else if (guess < secretNumber){printf("The number is higher than %d\n", guess);}
            else{printf("The number is lower than %d\n", guess);}
        }
        else{
            if (guess == secretNumber){printf("You guessed the number on your final guess! %d\n", secretNumber); loop = 0;}
            else{printf("You lose!");loop = 0;}
        }
    }
    return 0;
}